recipes/otwrapy/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.6" %}
+
+package:
+  name: otwrapy
+  version: {{ version }}
+
+source:
+  fn: otwrapy-{{ version }}.tar.gz
+  url: https://github.com/felipeam86/otwrapy/archive/{{ version }}.tar.gz
+  sha256: f7fb1c4a39a55b96bbe5c91c718b06b063223a2d2fb04442df3197fce47f50c4
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
+    - numpy
+    - joblib
+    - ipyparallel
+    - openturns
+  run:
+    - python
+    - numpy
+    - joblib
+    - ipyparallel
```

Comment:
Should these have the same version constraints that [source]( https://github.com/felipeam86/otwrapy/blob/0.6/setup.py#L25-L26 ) has?